tsconfig.json

```diff
@@ -35,6 +35,7 @@
       "node"
     ],
     "paths": {
+      "#src/*": ["./*"],
```

Comment:
did we resolve the problems that in new code IDE's use this for imports but then the references don't work properly?

Reply:
This now works with the angular builder. So if an app source file uses this path the builder resolves it properly. But please check that it works with your IDE as expected.